odoo/addons/base/models/ir_sequence.py

```diff
@@ -89,7 +89,7 @@ def _get_number_next_actual(self):
         '''Return number from ir_sequence row when no_gap implementation,
         and number from postgres sequence when standard implementation.'''
         for seq in self:
-            if seq.implementation != 'standard':
+            if seq.implementation != 'standard' or not isinstance(seq.id, int):
                 seq.number_next_actual = seq.number_next
             else:
                 seq_id = "%03d" % seq.id
```

Comment:
@CarlosRoca13 I'd say this would be a propper approach, as in this fix: https://github.com/odoo/odoo/pull/49701
```suggestion
            if seq.implementation != 'standard':
                seq.number_next_actual = seq.number_next
            else:
                seq_id = "%03d" % seq._origin.id
```